### PR TITLE
fix(sdk-core): choose correct default apiVersion for tss

### PR DESCRIPTION
**Pull request by zahin-mohammad**
TICKET: WP-3038

### CODE REVIEW

File: modules/bitgo/test/v2/unit/coins/utxo/prebuildAndSign.ts [hunk 1]
```diff
@@ -212,6 +218,7 @@ function run(coin: AbstractUtxoCoin, inputScripts: ScriptType[], txFormat: TxFor
 
     [true, false].forEach((useWebauthn) => {
       it(`should succeed with ${useWebauthn ? 'webauthn encryptedPrv' : 'encryptedPrv'}`, async function () {
+        const txCoins = ['tzec', 'zec', 'ltc', 'bcha', 'doge', 'dash', 'btg', 'bch'];
```
Comment (margueriteblair) [inline modules/bitgo/test/v2/unit/coins/utxo/prebuildAndSign.ts:221]: Should this not include `btc`?

File: modules/bitgo/test/v2/unit/coins/utxo/prebuildAndSign.ts [hunk 2]
```diff
@@ -257,6 +265,7 @@ function run(coin: AbstractUtxoCoin, inputScripts: ScriptType[], txFormat: TxFor
       it(`should be able to build, sign, & verify a replacement transaction with selfSend: ${selfSend}`, async function () {
         const rbfTxIds = ['tx-to-be-replaced'],
           feeMultiplier = 1.5;
+        const txCoins = ['tzec', 'zec', 'ltc', 'bcha', 'doge', 'dash', 'btg', 'bch'];
```
Comment (margueriteblair) [inline modules/bitgo/test/v2/unit/coins/utxo/prebuildAndSign.ts:268]: If we're gonna reuse this `txCoins`, can we make it global?

**Review by margueriteblair — COMMENTED** [review]
Most tests and changes look good - just a few nitty q's

**Review by OttoAllmendinger — APPROVED** [review]
not sure why the utxo change was necessary

**Comment by alebusse** [comment]
> not sure why the utxo change was necessary

Thanks for reviewing @OttoAllmendinger , the test change was necessary because the default wallet type is now 'hot' and it changes the prebuild params for utxos here
https://github.com/BitGo/BitGoJS/blob/98b8f31f9933cbf05716a1b83fb20b8e67925e3f/modules/abstract-utxo/src/abstractUtxoCoin.ts#L1084 and the nocks didnt match after that change.

**Review by alebusse — APPROVED** [review]

**Pull request merged**